pkg/utils/permission.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import "strings"
+
+var (
+	notFoundMsg         = "notfound"
```

Comment:
notfound is too general, I'm afraid some other cloud errors will reach here too. it's better to add another key word. such as `io`